samples/notebooks/dotnet/Getting-Started-Notebook.ipynb

```diff
@@ -3,14 +3,7 @@
   {
    "attachments": {},
    "cell_type": "markdown",
-   "metadata": {
-    "dotnet_interactive": {
-     "language": "csharp"
-    },
-    "polyglot_notebook": {
-     "kernelName": "csharp"
-    }
-   },
+   "metadata": {},
```

Comment:
was it intentional to clear out the metadata? seems to have occurred in several different places in the file.

Reply:
I usually do a "Clear All Outputs" for notebooks before I commit them on Github. It sometimes will get rid of this metadata especially for Polyglot notebooks, but it's not always consistent.